river/command/map.zig

```diff
@@ -74,7 +77,9 @@ pub fn map(
     // Check if the mapping already exists
     const mode_mappings = &seat.input_manager.server.config.modes.items[mode_id].mappings;
     for (mode_mappings.items) |existant_mapping| {
-        if (existant_mapping.modifiers == modifiers and existant_mapping.keysym == keysym and existant_mapping.release == optionals.release) {
+        if (existant_mapping.modifiers == modifiers
+            and existant_mapping.keysym == keysym
+            and existant_mapping.release == @field(options, "-release")) {
```

Comment:
Pretty sure you can just do this here:
```suggestion
            and existant_mapping.release == options.@"-release") {
```
`@field()` is only needed if the name isn't static at comptime.